relayer-cli/src/conclude.rs

```diff
@@ -96,6 +100,40 @@ impl Output {
         self
     }
 
+    /// Quick-access constructor for an output signalling a success `status` and tagged with the
+    /// input `res`.
+    pub fn success(res: impl Serialize + std::fmt::Debug) -> Self {
```

Comment:
@romac loved your suggestions so I implemented them.

Not sure if this approach to handle the `unwrap` is over-complicated (see the method `serialize_result` below). WDYT?